crates/api_common/src/utils.rs

```diff
@@ -1178,6 +1178,20 @@ fn build_proxied_image_url(
   ))
 }
 
+#[allow(clippy::expect_used)]
+pub fn check_code_verifier(code_verifier: &str) -> LemmyResult<()> {
```

Comment:
This is only used in create.rs so move it directly in that file, and make the function private.